pom.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.ibm.bamoe</groupId>
+    <artifactId>bamoe-maven</artifactId>
+    <version>9.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <name>BAMOE product Maven modules parent</name>
+
+    <properties>
+        <maven.compiler.source>17</maven.compiler.source>
```

Comment:
I'm not sure if we should push to 17.

I'd tackle the 17 from the docs perspective, but I'd be on 11 here

Reply:
Fixed.